runtime-display/mesa-pvr-ddk119/spec

```diff
@@ -0,0 +1,3 @@
+VER=22.1.3+vf2sdk3.8.2
+SRCS="git::commit=tags/mesa-22.1.3::http://gitlab.freedesktop.org/mesa/mesa.git"
+CHKSUMS="SKIP"
```

Comment:
CHKUPDATE here but note that it should not be updated without consulting SDK requirements.